package.json

```diff
@@ -12,15 +12,15 @@
     "test:watch": "node node_modules/.bin/jest --watchAll --config config/jest.config.js",
     "test:current": "node node_modules/.bin/jest --watch --config config/jest.config.js",
     "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "storybook": "cross-env NODE_ENV=storybook start-storybook -p 6006",
-    "build-storybook": "cross-env NODE_ENV=storybook NODE_OPTIONS=--max_old_space_size=8192 build-storybook",
+    "stories": "tfm-stories",
```

Comment:
what about your suggestion for `npm start`?

Reply:
IMO in foreman `npm start` should run the `webpack-dev-server`, and in plugins, it should run the `tfm-stories`.